internal/graph/memory.go

```diff
@@ -101,26 +117,32 @@ func (m *Memory) Predecessors(_ context.Context, node ocispec.Descriptor) ([]oci
 	return res, nil
 }
 
-// Remove removes the node from its predecessors and successors.
-func (m *Memory) Remove(ctx context.Context, node ocispec.Descriptor) error {
+// Remove removes the node from its predecessors and successors, and returns the
+// dangling nodes caused by the deletion.
```

Comment:
I think this description is more accurate.
```suggestion
// dangling root nodes caused by the deletion.
```

Reply:
changed accordingly.